application/src/main/java/dev/ikm/komet/app/aboutdialog/AboutDialogController.java

```diff
@@ -0,0 +1,50 @@
+package dev.ikm.komet.app.aboutdialog;
+
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.Label;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.URL;
+import java.time.Year;
+import java.util.ResourceBundle;
+
+public class AboutDialogController implements Initializable {
```

Comment:
Can you remove the `implements Initializable` interface?
Starting with Java 8 this dependency is no longer needed. It was required in JavaFX 2.0.
Tech Note: https://stackoverflow.com/questions/51392203/what-does-initialize-mean-in-javafx

Since you have the method with a prefix of the annotation `@FXML` FXML loader will call it appropriately.